components/prize_pool/commons/prize_pool_legacy.lua

```diff
@@ -171,14 +170,24 @@ function LegacyPrizePool.mapSlot(slot, mergeSlots)
 	local opponentsInSlot = #slot
 	Table.iter.forEachPair(CACHED_DATA.inputToId, function(parameter, newParameter)
 		local input = slot[parameter]
-		if newParameter == 'seed' then
+		if input == 'q' and slot.link then
+			-- Use qualifier display
+			if not slot.link:find('^%[%[') then
+				slot.link = '[[' .. slot.link .. ']]'
```

Comment:
You _could_ do, to maintain the checkmark whilst getting it count as a qualifier (related to @iMarbot comment)
```suggestion
				slot.link = '[[' .. slot.link .. '|'.. CHECKMARK ..']]'
```

Reply:
HTML is removed from the link display, so would require some more effort (e.g. not use handleSeed).

Reply:
https://github.com/Liquipedia/Lua-Modules/commit/44b3ee1aceb0e7e010c0c12a2a6531401b6eb2b9 Possible solution

Reply:
IMO, if these checkmarks are wanted, we should build support for these via the non-legacy prizepool with a switch to enable in case of direct qualifier tournaments.
Otherwise, people will start to use the legacy prize pools on purpose, just to get these checkmarks.

Reply:
Resolving this with #2333